.github/workflows/nightly.yml

```diff
@@ -31,117 +36,157 @@ jobs:
       uses: actions/checkout@v2
       with:
         path: wlan-testing
+        ref: staging-wifi-2047
```

Comment:
Shouldn't this be master?

Reply:
@jaspreetsachdev Think you have been looking at an old state of this file.